<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="credit_calc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>700</width>
      <height>65</height>
     </rect>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string> (</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>101</width>
      <height>76</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 212, 121, 107);</string>
    </property>
    <property name="text">
     <string>График</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_finish">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>213</green>
          <blue>109</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>208</red>
          <green>208</green>
          <blue>208</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="96">
          <red>164</red>
          <green>166</green>
          <blue>168</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_str">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>155</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>160</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="midLineWidth">
     <number>-2</number>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:30pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_e">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
